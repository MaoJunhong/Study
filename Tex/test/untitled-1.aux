\relax 
\@writefile{toc}{\contentsline {section}{\numberline {1} title }{2}}
\@writefile{toc}{\contentsline {section}{\numberline {2} title }{2}}
\@writefile{toc}{\contentsline {section}{\numberline {3} help }{2}}
