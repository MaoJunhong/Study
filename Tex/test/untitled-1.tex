\documentclass{article}
\usepackage{xeCJK}
\setCJKmainfont{宋体}
\setCJKmonofont{宋体}
\begin{document}
\tableofcontents
\newpage
\section{ title }
XXX is a SB. 我是中文
\section{ title }
XXX is a {Large big SB}, but smaller now.
%commit%

God says \begin{equation}a+b=c/(8+d)\end{equation}
$2^{32}, a_{12}$
\[
\lim_{x\to0}\frac{x}{\sin x}
\]
\section{ help }
file which contains all simbols use cmd: texdoc symbols-a4
\end{document}