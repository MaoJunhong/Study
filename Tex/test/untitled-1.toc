\contentsline {section}{\numberline {1} title }{2}
\contentsline {section}{\numberline {2} title }{2}
\contentsline {section}{\numberline {3} help }{2}
