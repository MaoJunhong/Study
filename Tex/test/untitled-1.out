\BOOKMARK [1][-]{section.1}{\040title\040}{}% 1
\BOOKMARK [1][-]{section.2}{\040title\040}{}% 2
